models/Discord/Commands/DiscordCommandCoder.js

```diff
@@ -38,7 +38,20 @@ class DiscordCommandCoder extends DiscordCommand {
         "https://cdn.discordapp.com/attachments/734475284446707753/772885346199011358/unknown.png",
         "https://cdn.discordapp.com/attachments/734475284446707753/774083235847077929/image0.png",
         "https://cdn.discordapp.com/attachments/734475284446707753/775784036470751232/image0.png",
-        "https://cdn.discordapp.com/attachments/734475284446707753/776741617053859850/unknown.png"
+        "https://cdn.discordapp.com/attachments/734475284446707753/776741617053859850/unknown.png",
+        "https://i.gyazo.com/566dd8a79f7aa374d7b248823d50a28a.png",
+        "https://cdn.discordapp.com/attachments/734475284446707753/734482071103471676/dreamseeker_2020-01-27_21-36-54.png",
+        "https://cdn.discordapp.com/attachments/734475284446707753/734508169178644500/ss13_alexkaraheal.PNG",
+        "https://cdn.discordapp.com/attachments/734475284446707753/734531895299276830/halfadoor.png",
```

Comment:
```suggestion
```
These aren't code or coder related

Reply:
relations
first: bood/jamie related
second: bug that caused the AI as a lone op to display as both being an AI and having TC purchases
third: paper being removed because it causes lag
fourth: alexcar
fifth: bug that causes half the door to be invisible